contracts/evoting/controller/action.go

```diff
@@ -357,10 +294,7 @@ func (a *scenarioTestAction) Execute(ctx node.Context) error {
 	if err != nil {
 		return xerrors.Errorf(getElectionErr, err)
 	}
-
-	dela.Logger.Info().Msg("Title of the election : " + election.Configuration.MainTitle)
-	dela.Logger.Info().Msg("ID of the election : " + string(election.ElectionID))
-	dela.Logger.Info().Msg("Status of the election : " + strconv.Itoa(int(election.Status)))
+	delaDebug(election.Configuration.MainTitle,string(election.ElectionID),strconv.Itoa(int(election.Status)))
```

Comment:
Here and for all the others below
```suggestion
	delaDebug(election.Configuration.MainTitle, string(election.ElectionID), strconv.Itoa(int(election.Status)))
```